### DISCUSSION TITLE
ProtonDrive Cloud Backup Support

### DISCUSSION THREAD

**Discussion by cuathro** [standardnotes/forum#2799] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes.

**Is your feature request related to a problem? Please describe.**
Currently, the only services enabled for cloud backup are services that have little or no respect for privacy (Google, Microsoft, Dropbox). Just having to log in by accepting the cookies of their services is already obnoxious, even more so when you try to protect yourself via VPN from their intrusive personal data collection clauses, and the only thing you get is a temporary blocking of your account for "suspicious activity". 

**Describe the solution you'd like**
One of the few companies that offer cloud services while respecting privacy is Proton, the same ones that developed the encrypted mail ProtonMail. In the last few months they enabled ProtonDrive, an encrypted storage system. Including ProtonDrive as a provider for encrypted cloud backups of encrypted notes not only sounds interesting but ultra-secure, and that's fascinating when you want to protect your freedom of thought (embodied in SN).

**Describe alternatives you've considered**
Simply add ProtonDrive as a cloud service provider for note backups.

**Additional context**

![Prototype_ProtonDrive-Intragration](https://user-images.githubusercontent.com/68512577/170719629-8857a943-240e-49bc-b3a3-81e6b32bc2a3.png)

**Comment by atmoio** [comment]
Cool mockup 👍 Most e2ee drive services shouldn't really have APIs, because that would break the e2ee model. What we'll likely transition to overtime is using the same system we use for File Backups: selecting a local folder for backups, which can happen to be your Dropbox, Google Drive, or Proton Drive.

**Comment by cuathro** [comment]
You are absolutely right. And the possibility of the same system of File Backups existing for Notes sounds great!
